### ISSUE TITLE
meta: tags in this repository don't match the ones on npmjs.com

### ISSUE THREAD

**Issue by Neurone** [shapeshift/hdwallet#406]
Tags in this repository - currently the latest is `v1.16.2` - don't match the ones on npmjs.com - currently latest `1.18.4`.

**Comment by mrnerdhair** [comment]
Yeah, true... our CI in this repo doesn't set tags and it should.
